<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnGestor</class>
 <widget class="QMainWindow" name="vtnGestor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor de Contenido Digital para Cursos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_registro">
       <attribute name="title">
        <string>Registro / Edición</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_datos_contenido">
          <property name="title">
           <string>Datos del Contenido</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_id">
             <property name="text">
              <string>ID (Solo Edición):</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txt_id">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lbl_titulo">
             <property name="text">
              <string>Título:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="txt_titulo"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lbl_fecha">
             <property name="text">
              <string>Fecha:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="date_fecha">
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lbl_autor">
             <property name="text">
              <string>Autor:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="txt_autor"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lbl_tipo_contenido">
             <property name="text">
              <string>Tipo de Contenido:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="cmb_tipo_contenido">
             <item>
              <property name="text">
               <string>Video</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Documento</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>EnlaceWeb</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_detalles_especificos">
             <property name="title">
              <string>Detalles Específicos</string>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="lbl_url_video">
                <property name="text">
                 <string>URL Video:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="txt_url_video"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lbl_duracion">
                <property name="text">
                 <string>Duración (min):</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="txt_duracion"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lbl_tipo_documento">
                <property name="text">
                 <string>Tipo Documento:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="txt_tipo_documento"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="lbl_tamano_documento">
                <property name="text">
                 <string>Tamaño (KB):</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="txt_tamano_documento"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="lbl_url_enlace">
                <property name="text">
                 <string>URL Enlace:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="txt_url_enlace"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_guardar">
            <property name="text">
             <string>Guardar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_limpiar">
            <property name="text">
             <string>Limpiar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_cancelar_edicion">
            <property name="text">
             <string>Cancelar Edición</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_consulta">
       <attribute name="title">
        <string>Consulta / Eliminación</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_busqueda">
          <property name="title">
           <string>Buscar Contenido</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="lbl_buscar">
             <property name="text">
              <string>Título a buscar:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_buscar"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_buscar">
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_mostrar_todo">
             <property name="text">
              <string>Mostrar Todo</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_contenidos">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Título</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Fecha</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Autor</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tipo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Detalles Específicos</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_editar_seleccionado">
            <property name="text">
             <string>Editar Seleccionado</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_eliminar_seleccionado">
            <property name="text">
             <string>Eliminar Seleccionado</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
